Notes/implementation-notes.md

```diff
@@ -129,7 +129,7 @@ require. It also tries to resolve anything defined by proof search.
 
 Additional type inference
 -------------------------
-A '?' in a type means "infer this part of the type".  This is distinct from "_"
+A '?' in a type means "infer this part of the type".  This is distinct from "\_"
 in types, which means "I don't care what this is". The distinction is in what
 happens when inference fails.  If inference fails for "_", we implicitly bind a
```

Comment:
There seem to be some others hanging around too! :)

Reply:
The others are not interpreted since they aren't paired. GitHub's diff hasn't the same behavior as the renderer: see [here](https://github.com/ether42/Blodwen/blob/master/Notes/implementation-notes.md#additional-type-inference) for result.

Reply:
It's more an issue if the text changes. These might be better off as inline code though, then you wouldn't have to worry about escaping them.

Reply:
Yeah, you are right, I shouldn't have been lazy :)